website/docs/getting-started/flashing-nrf52.md

```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
 
 ## Pre-requisits
 
-Please ensure that you use a data USB cable, as some cables provide power only and not the data lines.
+Please ensure that you use a data USB cable, as some cables provide power only and not the data lines. Many popular brands (ie: Anker) are also "power" only.
 
 ### T-Echo
 
+NOTE: The usb-C to usb-A cable from LILYGO is NOT a "data" cable, and can only be used for charging. 
+
+Many popular brands (ie: Anker) are also "power" only. 
+
+Confirm that you have a usb-c cable with "data" lines -- many cables (ie: The one that comes from LilyGo, Anker, etc are "power" only)
+
+#### Windows:
+
 You may need to install the [USB device drivers](http://www.wch-ic.com/search?q=ch340&t=downloads) if your device does not show up when connected.
 
+#### Mac OS
+
+(Verified as of 11/2021 on the latest T-Echo nRF52840 on Mac OS Monterey v12.0.1 - Intel chipset)
+
+If you have it, you can use the Apple (iPad Pro included) usb-C to usb-C cable, as it has "data" lines. The cable provided from LILYGO will not work, as it's power only.
+
+CRITICAL: *DO NOT* download the USB device drivers - this will actually prevent you from connecting to your T-Echo from your Mac.
+
+If you have already downloaded/installed the MacOS WCH-IC CH340 ("CH341SER_MAC") drivers via the `CH34x_Install_V1.5.pkg`, you will have to *uninstall* the kernel extension:
+
+* Unplug your T-Echo
+* Open the Terminal and run: `sudo -rf /Library/Extensions/usbserial.kext`
+* Reboot
+
+If you are starting clean (or have removed the WCH USB driver), verify successful connections with 
+
+* Plug in your T-Echo
+* Open the Terminal
+* `ls -l /dev/tty.usbmodem*`
+
+If the device file exists, you will also notice a "TECHOBOOT" volume in the Finder, or in the Terminal after double-clicking the Reset Button (see below) 
+
+* `ls /Volumes/TECHOBOOT`
```

Comment:
Might be a good case to use `details`: https://docusaurus.io/docs/markdown-features#details
for edge cases as users are most likely to want to read them if they are looking for it in the first place.

Reply:
Done - added detail block (overwrote latest push - see new file)